lib/active_model/serializer/array_serializer.rb

```diff
@@ -11,9 +11,8 @@ def initialize(resources, options = {})
         @root = options[:root]
         @object = resources
         @serializers = resources.map do |resource|
-          serializer_class = options.fetch(:serializer) do
-            ActiveModel::Serializer.serializer_for(resource)
-          end
+          serializer_context_class = options.fetch(:serializer_context_class, ActiveModel::Serializer)
+          serializer_class = options.fetch(:serializer) { serializer_context_class.serializer_for(resource) }
```

Comment:
Potential :bug: `serializer_context_class` is a required option.  Would you mind adding a comment to the initializer to that effect? and if it's absent, it'll throw a KeyError.  How do we want to handle that?  

Maybe just say "don't use ArraySerializer directly, Just use SerializableResource" :)